sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+- Updated this package to include package references to the now stable versions of `OpenTelemetry.Instrumentation.AspNetCore` and `OpenTelemetry.Instrumentation.Http`.
```

Comment:
Make it as different points on vendorded code removal and adding package references.

Reply:
updated. I tried to match the language we used in previous releases.